app/presenters/supergroups/news_and_communications.rb

```diff
@@ -36,9 +36,14 @@ def document_list(taxon_id)
     end
 
     def news_item_photo(base_path)
+      default_news_image = {
```

Comment:
Is this stored anywhere? Can the default image be retrieved from anywhere rather than being hard-coded here?

Reply:
Have adjusted to use relative URL: '/government/assets/placeholder.jpg'